job-service-acceptance-tests/pom.xml

```diff
@@ -474,6 +475,7 @@
                             <run>
                                 <ports>
                                     <port>${jobtracking-worker.adminport}:8081</port>
+                                    <port>5005:5005</port>
```

Comment:
Ridiculous - you're wasting my time submitting this - I'm not reviewing any further.

Reply:
apologies, these were changes that were made when getting the IT's to run locally and should never have been committed. I have removed these commits and rebuilt the branch on Jenkins.